include/aikido/constraint/uniform/ConstantSampler.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef AIKIDO_CONSTRAINT_CONSTANTSAMPLER_H_
+#define AIKIDO_CONSTRAINT_CONSTANTSAMPLER_H_
+#include "../../statespace/Rn.hpp"
+#include "../Sampleable.hpp"
+
+namespace aikido {
+namespace constraint {
+
+/// ConstantSampler for RealVectorStates.
+/// Stub sampler for WeldJoint or any fixed constant state space.
+class ConstantSampler
+  : public constraint::Sampleable
```

Comment:
Rename `ConstantSampler` to `RnConstantSampler` since this only works on an `Rn` state space.